README.md

```diff
@@ -60,6 +60,57 @@ export MARQUEZ_TIMEOUT_MS=5000
 
 ```
 
+### Common Calls and Examples
+There are two main ways to use the client:
+1. Set the namespace
+2. Supply the namespace for calls that require it
+
+#### Instantiating the client
+```mc = Client()```
+
+#### Set a Namespace for the client to use
+```mc.set_namespace(existing_namespace)```
+
+#### Create a Namespace
+``` 
+mc.create_namespace(namespace_name, 
+    "owner name",
+    "description of namespace")
+   ```
+
+#### Create a Job
+```
+mc.create_job(job_name=job_name,
```

Comment:
this example uses all named parameters, but the call to `create_namespace` does not. can we standardize in the example?

Reply:
No problem. Adjusted the code to use a more real-world example with inline values.

Reply:
hm, not sure this should've been marked resolved. this was not addressed:


```
mc.create_namespace("accounting_team", 
    "Gabriel G. Marquez",
    "Jobs and data sets related to accounting.")
```

The above are not named parameters, but the next example uses them:

```
mc.create_job(job_name="quarterly_summary",
    location="accounting_summary.py",
    description="generates the quarterly accounting report",
    input_dataset_urns=["urn:dataset:accounting_db:monthly_rollups_q2_2019"],
    output_dataset_urns=["urn:dataset:accounting_db:quarterly_reports_q2_2019"],
    namespace_name="accounting_team")
```